src/modules/importer.js

```diff
@@ -0,0 +1,120 @@
+import fs from 'fs';
+
+export default class Importer {
+  listen(eventEmitter, path, sync = false) {
+    this.eventEmitter = eventEmitter;
+
+    this.eventEmitter.on('changed', () => {
+      if (sync) {
+        console.log(this.importSync(path));
+      } else {
+        let result = this.import(path);
+
+        result.then(data => {
+          console.log(data);
+        });
+      }
+    });
+  }
+
+  import(path) {
+    return new Promise((resolve, reject) => {
+      fs.readdir(path, (err, files) => {
+        if (err) {
+          return reject(err);
+        }
+
+        let content = this.readFiles(path, files);
+        content.then(csv => {
+          return this.csv(csv);
+        }).then(data => {
+          resolve(data);
+        });
+      });
+    });
+  }
+
+  importSync(path) {
+    let files = fs.readdirSync(path);

```

Comment:
Please dont create variables, if they used only once

Reply:
I think this is improve readability... I guess this looks pretty unclear:
'return this.csvSync(this.readFilesSync(path, fs.readdirSync(path)))'
Tell me, If I wrong)